packages/server/src/api/muster/muster.postgres.controller.ts

```diff
@@ -38,7 +38,6 @@ class MusterPostgresCtr {
       const rosters = await getRosterWithUnitsAndEdipis(unitId, orgId);
```

Comment:
I don't think this is doing what you ultimately want it to do :/...yes you are finding folks "on roster" but these are only recent (i.e. the current roster they are on) and you have no idea if the user was on that roster for a given date. In order to know that you must query the `roster_history` table. You need that so you can make sure that they were on the roster for a given unit on a given date in order to determine if an observation is actually expected from them first.

Reply:
Please see: https://app.asana.com/0/1188940305683068/1200529438257363
Namely: "For this effort we will only use the roster table, not the roster history table."
This was discussed with Austin. They typically only look at very recent data.

Reply:
I see, right on, we may need to revisit this with him because if they want exports of this data too (like the original muster page) the queries will have to be changed then in order to be accurate for any timespan.

Reply:
Since there is no aggregation taking place over individuals to reach a unit level, but rather just filtering on a unit roster, I think for this initial PR it is fine to "ignore" the roster history table. 

It will be somewhat ambiguous to the user either way whether we include roster history, which means the time date range they chose is NOT the actual time date-range used for that user's calculations, or we exclude roster history, which will ensure the time date range matches, but that the user has a "low" value due to not being on the roster during that time-frame rather than through non-response. At present, I would say our users are "used to" the latter issue.